mlir/include/Catalyst/IR/CatalystOps.td

```diff
@@ -140,4 +140,28 @@ def PythonCallOp: Catalyst_Op<"pycallback",
   }];
 }
 
+def ActiveCallbackOp: Catalyst_Op<"activeCallbackCall",
```

Comment:
Minor, but since a callback has no definition is it not always a call? 
```suggestion
def ActiveCallbackOp: Catalyst_Op<"activeCallback",
```

Reply:
I don't like this. I know it is more verbose, but in my mind the `callback` is the function defined by the user. And a `callbackCall` is the call site to where this function is called.